helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml

```diff
@@ -90,6 +90,7 @@ spec:
           {{- end }}
     files:
       {{- include "sshFiles" . | nindent 6}}
+      {{- include "ntpFiles" . | nindent 6}}
```

Comment:
Do we need this for kubeadmconfigtemplate too?

Reply:
We also want to configure ntp on the worker nodes.

Reply:
sorry. i missed this line https://github.com/giantswarm/cluster-cloud-director/pull/101/files#diff-7188efb4466b71c05057ba8992c443f8c9ab6bc2e1d67049a72f6a323aae2bdcR128